bots/coniksbot/cmd/run.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"os"
+	"os/signal"
+
+	"github.com/coniks-sys/coniks-go/bots"
+	"github.com/spf13/cobra"
+)
+
+// runCmd represents the run command
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run a CONIKS bot instance",
+	Long: `Run a CONIKS bot instance
+
+This will look for config files with default names (botconfig.toml) 
+in the current directory if not specified differently.
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		config := cmd.Flag("config").Value.String()
+		run(config)
+	},
+}
+
+func init() {
+	RootCmd.AddCommand(runCmd)
+	runCmd.Flags().StringP("config", "c", "botconfig.toml", "Path to bot configuration file")
```

Comment:
Ah, you mean cause there's no configs to reload.  Ya, I guess it's unnecessary.  Please ignore.